signingscript/docker.d/init_worker.sh

```diff
@@ -28,16 +28,18 @@ export ZIPALIGN_PATH=/usr/bin/zipalign
 export PASSWORDS_PATH=$CONFIG_DIR/passwords.json
 export GPG_PUBKEY_PATH=$CONFIG_DIR/gpg_pubkey
 export WIDEVINE_CERT_PATH=$CONFIG_DIR/widevine.crt
-export AUTHENTICODE_TIMESTAMP_STYLE=null
+export AUTHENTICODE_TIMESTAMP_STYLE=old
```

Comment:
👍 I think it makes sense to keep testing the timestamps in staging, so yeah, this is a good change :)